### PR TITLE
feat: add a tokemak APR fetching service

### PR DESCRIPTION
**Pull request by gomesalexandre**
## Description

This adds a service for fetching the FOX tokemak APR.

Note that this is self-contained in plugins folder for now, but eventually, we'll be able to use this app-wide to get FOXy's APY. 

## Notice

- [x] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/CONTRIBUTING.md) guide?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

Partially tackles #1834

## Risk

None, this is isolated and not used as app code yet. 

## Testing

Test added for the hook, CI should be happy.

### CODE REVIEW

File: src/plugins/foxPage/hooks/useFoxyApr.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import { CHAIN_REFERENCE } from '@shapeshiftoss/caip'
+import { AxiosError } from 'axios'
+import axios from 'axios'
+import { getConfig } from 'config'
+import { useEffect, useState } from 'react'
+
+const TOKEMAK_STATS_URL = getConfig().REACT_APP_TOKEMAK_STATS_URL
+const TOKEMAK_TFOX_POOL_ADDRESS = '0x808d3e6b23516967ceae4f17a5f9038383ed5311'
```
[the remark this answers is not in the document]
Comment (gomesalexandre) [reply src/plugins/foxPage/hooks/useFoxyApr.ts:8]: Shouldn't, if there's an upgrade of the pool contract, we/Tokemak probably have bigger problems WRT migration!

**Review by elmutt — DISMISSED** [review]

File: sample.env [hunk 2]
```diff
@@ -36,3 +36,4 @@ REACT_APP_GEM_WYRE_SUPPORTED_COINS=https://api.gem.co/institutions/wyre/supporte
 REACT_APP_GEM_ASSET_LOGO=https://gem-widgets-assets.s3-us-west-2.amazonaws.com/currencies/crypto/
 REACT_APP_GEM_ENV=production
 REACT_APP_GEM_API_KEY=bb4164a72246dae1e03010d664d6cdae4e19b2554de02e3bf6c3cd30aa7e359e
+REACT_APP_TOKEMAK_STATS_URL=https://stats.tokemaklabs.com/
```
Comment (0xdef1cafe) [inline sample.env:39]: i've added this to fleek for you

**Review by 0xdef1cafe — CHANGES_REQUESTED** [review]
fix variable naming otherwise goodly

File: src/plugins/foxPage/hooks/useFoxyApr.ts [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  chainId: string
+  pools: TokemakPool[]
+}
+export const useFoxyApr = () => {
+  const [data, setData] = useState<string | null>(null)
+  const [error, setError] = useState<AxiosError>()
+  const [loaded, setLoaded] = useState<boolean>(false)
+
+  useEffect(() => {
+    const loadGovernanceData = async () => {
```
[the remark this answers is not in the document]
Comment (gomesalexandre) [reply src/plugins/foxPage/hooks/useFoxyApr.ts]: unwoopsed

**Review by 0xdef1cafe — APPROVED** [review]

**Pull request merged**